app-grid/app-grid-style.html

```diff
@@ -121,7 +121,8 @@
   <template>
     <style>
 
-      :host {
+      :host,
+      :host ::content {
```

Comment:
While I was working on the PR I discovered that using `::content` without `:root` works, but should it?

In the Polymer docs (link above) this can be seen:

> Under shady DOM, the `<content>` tag doesn't appear in the DOM tree. Styles are rewritten to remove the `::content` pseudo-element, **and any combinator immediately to the left of** `::content`.
>
> This implies:
>
> - You must have a selector to the left of the `::content` pseudo-element.
>
>   ```
>   :host ::content div
>    ```

Also, in some code samples there's this clarification:

```
/* styling elements distributed to content (via ::content) requires */
/* selecting the parent of the <content> element for compatibility with */
/* shady DOM . This can be :host or a wrapper element. */
```

I already have this change in my workspace, and the tests and demos all work as expected, but should I really submit this change?